### ISSUE TITLE
Implement SSR

### ISSUE THREAD

**Issue by robines** [Samfundet/Samfundet4#1509]
Having some form of SSR is an absolute requirement for the project. We need it for SEO and to use Open Graph meta tags (for sharing on social media).

Our current stack does not support SSR very well, but luckily for us, major changes are coming in the next version of React Router (v7), which seemingly makes adding SSR a breeze. See [this video](https://youtu.be/ZcwA0xt8FlQ?si=0CC0aFJ8tynAiT9C&t=1295) from React Conf, demoing the RR v7 SSR functionality.

Sadly [there is no release date set for v7 yet](https://github.com/remix-run/react-router/discussions/12011#discussioncomment-10688398), so we can't just sit and wait until it releases. From the linked comment:

>The recommendation is to use Remix v2 if you plan to use RRv7 as RRv7 will be similar to Remix v2 with the future flags enabled.

I therefore believe we should start testing a migration from RR to Remix.

**Comment by emilte** [comment]
Absolute requirement? Er det det?
Det blir et helt annet type prosjekt

**Comment by robines** [comment]
@emilte Ja, det er det. Vi kan ikke launche en ny nettside som ikke dukker opp ordentlig på Google, eller som ikke får opp bilde og tittel på arrangementer når man deler de på sosiale medier. Da vil vi raskt bli bedt om å bytte tilbake til Samf3 til det blir fikset.

Ser ikke hvordan det blir et helt annet prosjekt? Med RR v7 vil vi forhåpentligvis kunne veldig enkelt enable SSR på enkelte sider, feks. forsiden og arrangement-siden. Det som blir forskjellig er at man ikke da kan bare serve statiske filer for frontend, men det vil ikke ha noen reelle konsekvenser så det tenker jeg går fint.

**Comment by emilte** [comment]
Så lenge ITK er åpen for å kjøre det så. Tidligere synes jeg å huske at node/npm økosystem var utelukket på deres servere
